test/Engine/ProtoTest/Associative/Inheritance.cs

```diff
@@ -40,7 +40,7 @@ public void InheritanceTest03()
         }
 
         [Test]
-        [Category("DSDefinedClass")]
+        [Category("DSDefinedClass_Ignored_DoNotTestInheritance")]

```

Comment:
I'm sorry, if you want me to review this, I'm going to need a more intelligable diff than this!